modules/templates/util_render.go

```diff
@@ -118,22 +119,32 @@ func RenderIssueTitle(ctx context.Context, text string, metas map[string]string)
 }
 
 // RenderLabel renders a label
-func RenderLabel(ctx context.Context, label *issues_model.Label) template.HTML {
-	labelScope := label.ExclusiveScope()
+func RenderLabel(ctx context.Context, locale translation.Locale, label *issues_model.Label) template.HTML {
+	var (
+		description string
+		archivedCSS string
+		textColor   = "#111"
+		isArchived  = !label.ArchivedUnix.IsZero()
+		labelScope  = label.ExclusiveScope()
+	)
 
-	textColor := "#111"
 	r, g, b := util.HexToRBGColor(label.Color)
 	// Determine if label text should be light or dark to be readable on background color
 	if util.UseLightTextOnBackground(r, g, b) {
 		textColor = "#eee"
 	}
 
-	description := emoji.ReplaceAliases(template.HTMLEscapeString(label.Description))
+	if isArchived {
+		description = locale.TrString("archived")
```

Comment:
The old description does not become obsolete just because the label was archived. The description is maybe still of interest, so it should not be replaced, but extended.